### ISSUE TITLE
Requires heterozygosity

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7685]
**Preferred term label:**
Requires heterozygosity

**Synonyms**

**Definition (free text, please give PubMed ID)**
Covers rare instances of a condition that is most severe in the heterozygous state. Such disorders are rare and currently all are X-linked. Most X-linked recessive conditions manifest if hemizygous in males, or biallelic in females, though may have a mild phenotype in the heterozygous state in females. However, Craniofrontonasal dysplasia due to EFNB1 and  PCDH19-related epilepsy are both X-linked recessive and paradoxically more severe in females. Hemizygous males may be mildly affected but seldom manifest the full phenotype. Importantly the mutant allele can be inherited from a normal or very mildly affected father.  The mechanism is currently accepted to be due to cellular interference whereby the two distinct cell populations (those with and without the variant) exhibit abnormal cellular interactions in the mosaic state - in women, who are functionally mosaic due to random X inactivation, or mosaic males. The same mechanism could theoretically be applicable to autosomal genes with a mosaic variant.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@angharadroberts - reworded the definition slightly as follows. It was added as HP:0034343

A rarely encountered phenomenon in which condition is most severe in the heterozygous state. Such disorders are rare and currently all are X-linked. Most X-linked recessive conditions manifest if hemizygous in males, or biallelic in females, though may have a mild phenotype in the heterozygous state in females. However, Craniofrontonasal dysplasia due to EFNB1 and  PCDH19-related epilepsy are both X-linked recessive and paradoxically more severe in females. Hemizygous males may be mildly affected but seldom manifest the full phenotype. Importantly the mutant allele can be inherited from a normal or very mildly affected father.  The mechanism is currently accepted to be due to cellular interference whereby the two distinct cell populations (those with and without the variant) exhibit abnormal cellular interactions in the mosaic state - in women, who are functionally mosaic due to random X inactivation, or mosaic males. The same mechanism could theoretically be applicable to autosomal genes with a mosaic variant.
